doc/building.rst

```diff
@@ -69,17 +77,40 @@ CMake project definition looks as follows:
     #   find_package(open62541 REQUIRED)
     #   target_link_libraries(main open62541::open62541)
 
-Building with CMake on Windows
+Building with Visual Studio on Windows
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Here we explain the build process for Visual Studio (2013 or newer). To build
-with MinGW, just replace the compiler selection in the call to CMake.
+Here we explain the build process for Visual Studio 2022 Community.
 
 - Download and install
+    - Python 3.x: https://python.org/downloads
+    - Visual Studio 2022: https://visualstudio.microsoft.com
+        - When installing Visual Studio select the workload "Desktop development with C++"
+
+- Open Visual Studio and from the Get started window select "Clone a repository"
+    - Repository location: https://github.com/open62541/open62541.git
+    - Select a local path where to download the project and press Clone
+- Switch to Folder View from the Solution Explorer
+- Project / CMake Settings for open62541
+        - Customize CMake variables as wanted and save
+- Build / Build All
+- Build / Install open62541
+
+Note: the solution generated with cmake can also be compiled in parallel with the command
+
+.. code-block:: bash
 
-  - Python 3.x: https://python.org/downloads
-  - CMake: http://www.cmake.org/cmake/resources/software.html
-  - Microsoft Visual Studio: https://www.visualstudio.com/products/visual-studio-community-vs
+    msbuild open62541.sln /v:n -t:rebuild -m
+
+Building with CMake/MinGW on Windows
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Ditto.

Reply:
good catch, I'm new to Sphinx and I misssed that